### PR TITLE
Migrated to use a function to get orientation rather than a static prop

### LINKED ISSUE #3: Usage in NextJS
Currently, due to the check in the initial code block, we're unable to use `o9n` in SSR projects like NextJS without having an error thrown. I would love to see `createOrientation` and a new `getOrientation` function exported for usage in SSR utilities.

I'm willing to take on development of this work, I just wanted to raise it as an issue to spark initial discussion

### PR DESCRIPTION
**Pull request by crutchcorn**
This should fix #3 without introducing any breaking changes.

I am using a (slightly, due to types) modified version of exactly this in my NextJS project using TypeScript (which is how I was able to catch some of the changes I needed to make)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -202,36 +211,36 @@ function getOrientationChangeEvent (name, props) {
...
-    addEventListener: function addEventListener (evt, cb) {
+    addEventListener: function addEventListener(evt, cb) {
       if (!this.listeners[evt]) {
         this.listeners[evt] = [];
       }
       if (this.listeners[evt].indexOf(cb) === -1) {
         this.listeners[evt].push(cb);
       }
     },
-    dispatchEvent: function dispatchEvent (evt) {
```
Comment (chmanie) [inline index.js]: I feel like I should introduce some code linting for this project

Comment (crutchcorn) [reply index.js]: Code linting would be of great assistance! I don't mind opening the PR to introduce code linting if you just want to point me towards what set of rules you'd like to configure :)

**Comment by crutchcorn** [comment]
@chmanie if you're interested, I can make a PR after this once is merge to rewrite the library into a modern compiled TypeScript iteration :)

File: index.js (hunk 1 above)
Comment (chmanie) [reply index.js:215]: Cool, that would be fantastic! Sorry for the long delay, this has been a busy time.
I think I’m good with just the prettier/eslint default rules except for using single quotes :)

**Comment by chmanie** [comment]
>  if you're interested, I can make a PR after this once is merge to rewrite the library into a modern compiled TypeScript iteration :)

That would be truly wonderful! Feel free to open a PR, this is highly appreciated!

**Comment by chmanie** [comment]
@crutchcorn I have added you as a collaborator to this project. In case you'd like to open more PRs, that's going to be a bit easier :)

**Comment by crutchcorn** [comment]
Thank you so much @chmanie! I'll start working on the linting and the TypeScipt ports soon enough

**Pull request merged**